packages/sit-onyx/src/components/OnyxRadioButton/OnyxRadioButton.vue

```diff
@@ -40,7 +48,34 @@ watchEffect(() => selectorRef.value?.setCustomValidity(props.errorMessage ?? "")
 </template>
 
 <style lang="scss">
-$input-size: var(--onyx-spacing-md);
+@use "../../styles/density.scss";
+
+.onyx-radio-button,
+.onyx-radio-button-skeleton {
+  @include density.compact {
+    --onyx-radio-button-height: var(--onyx-density);
+    --onyx-radio-button-selector-size: var(--onyx-spacing-sm);
+    --onyx-radio-button-dot-size: var(--onyx-spacing-4xs);
+    --onyx-radio-button-label-padding: var(--onyx-spacing-4xs);
+    --onyx-radio-button-selector-margin: var(--onyx-spacing-xs);
```

Comment:
```suggestion
    --button-height: var(--onyx-density);
    --selector-size: var(--onyx-spacing-sm);
    --dot-size: var(--onyx-spacing-4xs);
    --label-padding: var(--onyx-spacing-4xs);
    --selector-margin: var(--onyx-spacing-xs);
```

(optional) Personally I am not a huge fan of such long variable names :)

Reply:
I am worried that we will run into naming collisions, when we build more complex components